.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+## Why was this change made?
+
+
+
+## Was the API documentation (openapi.json) updated?
```

Comment:
Does Hydrus have API documentation or is this copy :spaghetti: ?

Reply:
oops - fixed.